src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -61,7 +61,15 @@ public static void main(String[] args) throws Exception {
         // create the command line parser
         CommandLineParser parser = new DefaultParser();
 
-        XMLConfiguration pluginConfig = buildConfiguration("config/plugin.xml");
+        // load the plugin configuration from the jar's folder or if not found the the $cwd/config
+        String jarDirPath = new File(DBWorkload.class.getProtectionDomain().getCodeSource().getLocation().toURI()).getParent();
+        File pluginFile = new File(jarDirPath + "/config/plugin.xml");
+        XMLConfiguration pluginConfig;
+        if (pluginFile.exists()) {
+            pluginConfig = buildConfiguration(jarDirPath + "/config/plugin.xml");
+        } else {
+            pluginConfig = buildConfiguration("config/plugin.xml");
+        }
```

Comment:
I suggest reversing this logic.
Current state of affairs is to load the data from the current directory, so we should default to that to avoid breaking existing installs that might be doing something clever.
If that *doesn't* exist, then we should fallback to loading it from the jar directory.